lib/mpoolMonitor/mpool_monitor.go

```diff
@@ -0,0 +1,180 @@
+package mpoolMonitor
```

Comment:
nit: usually directory names and package names are all lowercase (eg remoteblockstore)